Detectors/Vertexing/src/SVertexer.cxx

```diff
@@ -200,9 +200,38 @@ void SVertexer::process(const o2::globaltracking::RecoContainer& recoData, o2::f
         }
       }
     }
-    if (mStrTracker->getMCTruthOn()) {
-      auto& strTrMCLableOut = pc.outputs().make<std::vector<o2::MCCompLabel>>(o2f::Output{"GLO", "STRANGETRACKS_MC", 0, o2f::Lifetime::Timeframe});
-      strTrMCLableOut.swap(mcLabsOut);
+
```

Comment:
pmr::vectors are more expensive, so it is better to avoid using them as intermediate containers. 
Could you do the https://github.com/AliceO2Group/AliceO2/blob/ede780154a9c4b09519338d619feb8afedecbe6c/Detectors/Vertexing/src/SVertexer.cxx#L181-L202 accumulation in the usual std::vectors 
and then do the reassignment https://github.com/AliceO2Group/AliceO2/blob/ede780154a9c4b09519338d619feb8afedecbe6c/Detectors/Vertexing/src/SVertexer.cxx#L216-L222 directly from this temporary std::vec to strTracksOut etc.
Then the https://github.com/AliceO2Group/AliceO2/blob/ede780154a9c4b09519338d619feb8afedecbe6c/Detectors/Vertexing/src/SVertexer.cxx#L224-L228 is not needed. Also, I would not care about avoiding this reassignment in case of a single thread, though one can obviously skip the sorting in this case.